lib/profile/ProfilesController.js

```diff
@@ -157,6 +159,53 @@ class ProfilesController extends BaseController {
       done(null, cmdResult);
     });
   }
+
+  login(options, done) {
+    const activeProfile = this.cliManager.getActiveProfile();
+    const profileName = options.name || (activeProfile && activeProfile.name);
+    if (isNullOrUndefined(profileName)) {
+      return setImmediate(() => { done(getItemError(EntityType.PROFILE)); });
+    }
+
+    if (!this.cliManager.profileExists(profileName)) {
+      const err = new KinveyError(Errors.ProfileNotFound);
+      return done(err);
+    }
+
+    async.waterfall([
+      (next) => {
+        const q = this.cliManager.prompter.buildQuestion(
```

Comment:
I am not sure what it displays exactly, but I think we should have the email in the prompt somewhere.
Something like:
Email: email@example.com
Password:

Reply:
We already have it in the profile :confused:

Reply:
I mean, not ask for it, but show it, so that the customer knows which password he needs to write.